x/ratesync/keeper/hooks.go

```diff
@@ -70,7 +70,7 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 	for _, hc := range hcs {
 		if hc.Features.LiquidStake.Enabled && epochIdentifier == types.LiquidStakeEpoch {
 			// Add liquidstakekeeper and do stuff
-			err := k.ExecuteLiquidStakeRateTx(ctx, hc.Features.LiquidStakeIBC, liquidBondDenom, bondDenom, nas.MintRate, hc.ID, hc.ConnectionID, hc.ICAAccount)
+			err := k.ExecuteLiquidStakeRateTx(ctx, hc.Features.LiquidStake, liquidBondDenom, bondDenom, nas.MintRate, hc.ID, hc.ConnectionID, hc.ICAAccount)
```

Comment:
> take LiquidStakeIBC feature instead of LiquidStake

but it's taking LiquidStake instead of LiquidStakeIBC?

Reply:
yea, it is inside `LiquidStake.Enabled` , so the features have to be `hc.Features.LiquidStake`

Reply:
oh my bad, changelog is bad